include/envoy/stream_info/filter_state.h

```diff
@@ -55,6 +55,69 @@ class FilterState {
             (dynamic_cast<const T*>(getDataGeneric(data_name)) != nullptr));
   }
 
+  /**
+   * The addToList, hasListWithName, forEachListItem are variants of the above
+   * functions, that operate on list data. Multiple elements could be added
+   * the list under the same data_name.
+   * @param data_name the name of the data being added to the end of a list.
```

Comment:
Nit: The "to the end of the list" phrasing implies to me that the data within a list may have different names; could you clarify that?

Reply:
It just means that the data is being appended to the list object attached to the key (data_name)

Reply:
Oh sorry.. I see what you are saying.. Its copy paste left over.